docs/src/main/sphinx/installation/containers.md

```diff
@@ -88,6 +88,18 @@ at `/etc/trino/catalog`, or individual catalog property files in it.
 
 If you want to use additional plugins, mount them at `/usr/lib/trino/plugin`.
 
+To avoid having to create catalog files and mount them in the container,
+dynamic catalog management can be enabled by setting the `CATALOG_MANAGEMENT`
+environmental variable to `dynamic`.
+
+```text
+$ docker run --name trino -d -p 8080:8080 -e CATALOG_MANAGEMENT=dynamic trinodb/trino
+```
+
+After connecting to Trino, execute [](/sql/create-catalog) statements to add
```

Comment:
Reword and link to https://trino.io/docs/current/language/sql-support.html#catalog-management with the anchor in the code

Reply:
Fixed